### PR TITLE
Staging v1.12.0 beta.1 (#1771)

### PR DESCRIPTION
**Pull request by ddimatos**
As part of the release work for version 1.12.0-beta.1 where content may have diverged after branching from `dev`; this PR cherry picks those divergent changes back into the development branch `dev`. 

The cherry pick can bee seen [here](https://github.com/ansible-collections/ibm_zos_core/commit/2b87759b84547abecf9fec5d652a86792b0cd174).

### CODE REVIEW

**Comment by ddimatos** [comment]
tests/functional/modules/test_zos_backup_restore.py
image

**Comment by ddimatos** [comment]
tests/functional/modules/test_zos_blockinfile_func.py
image

**Comment by ddimatos** [comment]
tests/functional/modules/test_zos_copy_func.py
TBD

**Comment by ddimatos** [comment]
tests/functional/modules/test_zos_fetch_func.py 
image

**Comment by ddimatos** [comment]
tests/functional/modules/test_zos_job_output_func.py
image

**Comment by ddimatos** [comment]
tests/functional/modules/test_zos_job_query_func.py
image

**Comment by ddimatos** [comment]
tests/functional/modules/test_zos_job_submit_func.py
image

**Comment by ddimatos** [comment]
tests/functional/modules/test_zos_backup_restore.py
tbd

File: docs/source/modules/zos_job_output.rst [hunk 1]
```diff
@@ -265,7 +265,7 @@ jobs
     | **type**: str
 
   content_type
-    Type of address space used by the job.
+    Type of address space used by the job, can be one of the following types. - APPC for a APPC Initiator. - JGRP for a JOBGROUP. - JOB for a Batch job. - STC for a Started task. - TSU for a Time sharing user. - \? for an unknown or pending.
```
Comment (richp405) [inline docs/source/modules/zos_job_output.rst:268]: quibble: shouldn't it be "APPC for an APPC Initiator"?

File: docs/source/modules/zos_job_query.rst [hunk 2]
```diff
@@ -184,11 +184,35 @@ jobs
     | **sample**: JOB01427
 
   content_type
-    Type of address space used by the job.
+    Type of address space used by the job, can be one of the following types.
+
+    APPC for a APPC Initiator.
```
Comment (richp405) [inline docs/source/modules/zos_job_query.rst:189]: Same quibble.

File: docs/source/modules/zos_job_submit.rst [hunk 3]
```diff
@@ -593,7 +593,19 @@ jobs
     | **sample**: HELLO
 
   content_type
-    Type of address space used by the job.
+    Type of address space used by the job, can be one of the following types.
+
+    APPC for a APPC Initiator.
```
Comment (richp405) [inline docs/source/modules/zos_job_submit.rst:598]: An, not A.

File: plugins/modules/zos_job_output.py [hunk 4]
```diff
@@ -113,7 +113,13 @@
       sample:
     content_type:
       description:
-         Type of address space used by the job.
+        Type of address space used by the job, can be one of the following types.
+        - APPC for a APPC Initiator.
```
Comment (richp405) [inline plugins/modules/zos_job_output.py:117]: An

File: plugins/modules/zos_job_query.py [hunk 5]
```diff
@@ -121,7 +121,13 @@
       sample: JOB01427
     content_type:
       description:
-         Type of address space used by the job.
+        - Type of address space used by the job, can be one of the following types.
+        - APPC for a APPC Initiator.
```
Comment (richp405) [inline plugins/modules/zos_job_query.py:125]: An

File: plugins/modules/zos_job_submit.py [hunk 6]
```diff
@@ -165,7 +165,13 @@
       sample: HELLO
     content_type:
       description:
-         Type of address space used by the job.
+        - Type of address space used by the job, can be one of the following types.
+        - APPC for a APPC Initiator.
```
Comment (richp405) [inline plugins/modules/zos_job_submit.py:169]: An

File: tests/functional/modules/test_zos_job_submit_func.py [hunk 7]
```diff
@@ -446,9 +446,6 @@ def test_job_submit_pds_special_characters(ansible_zos_module):
         results = hosts.all.zos_data_set(
             name=data_set_name_special_chars, state="present", type="pds", replace=True
         )
-        hosts.all.shell(
-            cmd="echo {0} > {1}/SAMPLE".format(quote(JCL_FILE_CONTENTS), temp_path)
```
Comment (richp405) [inline tests/functional/modules/test_zos_job_submit_func.py:450]: Is this the line you were asking about being a duplicate?  It looks like the second echo call is being deleted here.

**Review by richp405 — APPROVED** [review]
I kept finding a grammatical issue (a versus an APPC Initiator), but nothing really programmatic, except the duplicate you mentioned perviously.

File: tests/functional/modules/test_zos_job_submit_func.py (hunk 7 above)
Comment (ddimatos) [reply tests/functional/modules/test_zos_job_submit_func.py:450]: Technically it was a triplicate as the merge was being managed by Git, 2 in dev and one in my merge, I limited it 1 and tested and regression passed.

File: docs/source/modules/zos_job_output.rst (hunk 1 above)
Comment (ddimatos) [reply docs/source/modules/zos_job_output.rst]: Done

File: docs/source/modules/zos_job_submit.rst (hunk 3 above)
Comment (ddimatos) [reply docs/source/modules/zos_job_submit.rst]: Done

File: plugins/modules/zos_job_query.py (hunk 5 above)
Comment (ddimatos) [reply plugins/modules/zos_job_query.py]: Done

File: plugins/modules/zos_job_submit.py (hunk 6 above)
Comment (ddimatos) [reply plugins/modules/zos_job_submit.py]: Done

File: plugins/modules/zos_job_output.py (hunk 4 above)
Comment (ddimatos) [reply plugins/modules/zos_job_output.py]: Done

File: docs/source/modules/zos_job_query.rst (hunk 2 above)
Comment (ddimatos) [reply docs/source/modules/zos_job_query.rst]: Done

**Review by fernandofloresg — APPROVED** [review]

**Review by ketankelkar — CHANGES_REQUESTED** [review]

**Pull request merged**